homeassistant/components/sensor/rest.py

```diff
@@ -33,6 +33,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     resource = config.get('resource', None)
     method = config.get('method', DEFAULT_METHOD)
     payload = config.get('payload', None)
+    verify_ssl = config.get('verify_ssl', False)
```

Comment:
Ah snap, caught this too late.

From the [requests docs](http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification):
> By default, verify is set to `True`.

We should default to `True` as well.

I'll update this in the dev branch.